core/pom.xml

```diff
@@ -49,6 +49,11 @@
             <artifactId>js</artifactId>
             <version>1.7R1</version>
         </dependency>
+        <dependency>
+            <groupId>net.jcip</groupId>
+            <artifactId>jcip-annotations</artifactId>
+            <version>1.0</version>
```

Comment:
As it is only used in test classes can you make this `<scope>test</scope>`.
Otherwise it will be on the classpath of every JNLP application started with icedtea-web

Reply:
Good idea.